src/pages/waiting-room-to-car/waiting-room-to-car.html

```diff
@@ -9,106 +9,191 @@
 
 <ion-content no-bounce>
   <div>
-    <form>
+    <form [formGroup]="form" (ngSubmit)="onSubmit()">
       <ion-grid>
-        <ion-row class="mes-full-width-card" id="eyesight-test-card" align-items-center>
-          <ion-col col-32>
+      <ion-row class="mes-validated-row" id="eyesight-test-card" align-items-center>
+          <div class="row-validation-bar" [class.ng-invalid]="isCtrlDirtyAndInvalid('eyesightCtrl')"></div>
+          <ion-col class="component-label" col-32>                
             <label>Eyesight test</label>
-          </ion-col>
-          <ion-col col-23>
-            <input type="radio" name="eyesight" id="eyesight-pass" class="gds-radio-button" (click)="eyesightPassPressed()" [checked]="pageState.eyesightPassRadioChecked$ | async">
-            <label for="eyesight-pass" class="radio-label">Pass</label>
-          </ion-col>
-          <ion-col>
-            <input type="radio" name="eyesight" id="eyesight-fail" class="gds-radio-button" (click)="eyesightFailPressed()" [checked]="pageState.eyesightFailRadioChecked$ | async">
-            <label for="eyesight-fail" class="radio-label">Fail</label>
-          </ion-col>
-        </ion-row>
+        </ion-col>
+        <ion-col>
+          <ion-row class="spacing-row"></ion-row>
+          <ion-row class="component-row" align-items-centre justify-content-start >
+              <ion-col col-32>
+                  <input type="radio" formControlName="eyesightCtrl" name="eyesightCtrl" id="eyesight-pass"
+                  class="gds-radio-button" [class.ng-invalid]="isCtrlDirtyAndInvalid('eyesightCtrl')"
+                  (click)="eyesightPassPressed()" [checked]="pageState.eyesightPassRadioChecked$ | async" value="P">
+                <label for="eyesight-pass" class="radio-label">Pass</label>
+              </ion-col>
+              <ion-col col-32>
+                  <input type="radio" formControlName="eyesightCtrl" name="eyesightCtrl" id="eyesight-fail"
+                  class="gds-radio-button" [class.ng-invalid]="isCtrlDirtyAndInvalid('eyesightCtrl')"
+                  (click)="eyesightFailPressed()" [checked] = "pageState.eyesightFailRadioChecked$ | async" value="F">
```

Comment:
Please remove spacing around assignment operator in template directives to make consistent with everywhere else.

Reply:
removed